packages/web-twig/src/Resources/components/FileUploader/stories/FileUploaderAttachmentWithImagePreview.twig

```diff
@@ -0,0 +1,48 @@
+<section class="docs-Section">
+    <h2 class="docs-Heading">Attachment with Image Preview</h2>
+
+    <Modal id="example_image_preview" titleId="example_image_preview_title">
+        <ModalDialog>
+            <ModalHeader modalId="example_image_preview" titleId="example_image_preview_title">
+                Image Preview
+            </ModalHeader>
+            <ModalBody>
+
+            </ModalBody>
+            <ModalFooter>
+                <Button
+                    color="primary"
+                    data-spirit-target="#example_image_preview"
+                    data-spirit-dismiss="modal"
+                >
+                    Confirm
+                </Button>
+                <Button
+                    color="secondary"
+                    data-spirit-target="#example_image_preview"
+                    data-element="cancel"
+                >
+                    Cancel
+                </Button>
+            </ModalFooter>
+        </ModalDialog>
+    </Modal>
+
+
+    <FileUploader data-spirit-toggle="fileUploader" id="example_customImagePreview">
+        <template data-spirit-snippet="item">
+            <FileUploaderAttachment onEdit="editHandler()" hasImagePreview />
+        </template>
+        <FileUploaderInput
+            helperText="Max size of each file is 10 MB"
+            id="fileUploaderWithImagePreview"
+            label="Label"
+            multiple
+            name="fileUploaderWithImagePreview"
+        />
+        <FileUploaderList headingId="attachments-customImagePreview" />
+    </FileUploader>
+
+    {{ encore_entry_script_tags('fileuploaderImagePreview') }}
```

Comment:
I couldn't get over this, even with `make build` 🙁. What steps are necessary?

![image](https://github.com/lmc-eu/spirit-design-system/assets/5614085/8874d89a-abb0-4529-9f8a-42812f4ea4ca)

Reply:
Try to delete docker images and rebuild or at least remove `build/` directory created by the Webpack in the demo app and run `make encore-build` in `packages/web-twig`.